tox.ini

```diff
@@ -50,17 +50,25 @@ allowlist_externals =
 [testenv:formatting]
 deps =
     flake8
-    black>=21.8b0
+    black>=21.8b0, <23.11
 commands =
     flake8 sambacc tests
     black --check -v .
 
 [testenv:schemacheck]
 deps =
-    black
+    black>=21.8b0, <23.11
     PyYAML
 commands =
     python -m sambacc.schema.tool
+# The following env var is a workaround for running this testenv on fedora 39
+# with python 3.12.1. As of approx. 2023-12-11 the ci started # to fail because
+# of the multidict library (not even one of our direct dependencies) failing
+# to install from source because we (intentionally) have no C compiler in the
+# image. Try to remove this in a few weeks to see if wheel availability or
+# whatever has been resolved.
+setenv =
+    MULTIDICT_NO_EXTENSIONS=1
```

Comment:
From recent CI runs I could see failures with Fedora 38 too. Is that also related to 'multidict' library?

Reply:
Black formatting. This pr has a temorary fix for that until I take a look at what changed in the recent releases of black.

Reply:
Ah..Ok.